subprojects/docs/src/snippets/groovy/compilationAvoidance/groovy/build.gradle

```diff
@@ -1,8 +1,5 @@
 subprojects {
     apply plugin: 'groovy'
-    repositories {
-        jcenter()
-    }
```

Comment:
:x: Are all the changes to "compilationAvoidance" in this PR expected. I think that snippet is already fixed in another PR?

Reply:
Slipped in accidentally. Fixed now.